narwhal/primary/src/synchronizer.rs

```diff
@@ -745,12 +752,13 @@ impl Synchronizer {
         let (sender, receiver) = oneshot::channel();
         self.inner
             .tx_certificate_acceptor
-            .send((certificate, sender, early_suspend))
+            .send((certificate.clone(), sender, early_suspend))
```

Comment:
It will be useful to assert that the certificate is in a verified state at the receiving end of `rx_certificate_acceptor`.